mammoth/tests/pull_request_chk.sh

```diff
@@ -115,49 +115,20 @@ ${PYTHON} onmt/bin/train.py \
 [ "$?" -eq 0 ] || error_exit
 echo "Succeeded" | tee -a ${LOG_FILE}
 
-echo -n "  [+] Testing NMT training w/ copy..."
-${PYTHON} onmt/bin/train.py \
-            -config ${DATA_DIR}/data.yaml \
-            -src_vocab $TMP_OUT_DIR/onmt.vocab.src \
-            -tgt_vocab $TMP_OUT_DIR/onmt.vocab.tgt \
-            -src_vocab_size 1000 \
-            -tgt_vocab_size 1000 \
-            -rnn_size 2 -batch_size 10 \
-            -word_vec_size 5 -report_every 5        \
-            -rnn_size 10 -train_steps 10 \
-            -copy_attn >> ${LOG_FILE} 2>&1
-[ "$?" -eq 0 ] || error_exit
-echo "Succeeded" | tee -a ${LOG_FILE}
-
```

Comment:
would be great to have stg of the sort, even if minimal

Reply:
There is no copy attention anymore.

We should add a few new cases here, though.